packages/flutter/test/services/platform_views_test.dart

```diff
@@ -276,4 +276,82 @@ void main() {
       );
     });
   });
+
+  group('Linux', () {
+    late FakeLinuxPlatformViewsController viewsController;
+    setUp(() {
+      viewsController = FakeLinuxPlatformViewsController();
+    });
+
+    test('create Linux view of unregistered type', () async {
+      expect(
+        () {
+          return PlatformViewsService.initUiKitView(
+            id: 0,
+            viewType: 'web',
```

Comment:
Not sure what this field is, but probably shouldn't be `web`?

Reply:
This test is to make sure that the platform views service should reject unregistered view type. Here, the view type may be any string literal.

Reply:
Done.